src/main/java/frc/robot/subsystems/swerve/Swerve.java

```diff
@@ -92,26 +99,63 @@ public Swerve() {
 
     rotPid.enableContinuousInput(-Math.PI, Math.PI);
 
-    driveRoutine = new SysIdRoutine(
-      new SysIdRoutine.Config(null, null, null, (state) -> Logger.recordOutput("SysIdTestState", state.toString())),
+    driveRoutine = setDriveRoutine();
+
+    angularRoutine = setAngularRoutine();
+  }
+
+  private SysIdRoutine setDriveRoutine() {
+    return new SysIdRoutine(
+      new SysIdRoutine.Config(
+        null, 
+        null, 
+        null, 
+        (state) -> Logger.recordOutput("SysIdTestState", state.toString())
+      ),
       new SysIdRoutine.Mechanism((Voltage v) -> {
-        modules[0].steer.setAngle(new Rotation2d());
-        modules[1].steer.setAngle(new Rotation2d());
-        modules[2].steer.setAngle(new Rotation2d());
-        modules[3].steer.setAngle(new Rotation2d());
+        for (SwerveModule module : modules) {
+          module.steer.setAngle(new Rotation2d());
+        }
+        // modules[0].steer.setAngle(new Rotation2d());
+        // modules[1].steer.setAngle(new Rotation2d());
+        // modules[2].steer.setAngle(new Rotation2d());
+        // modules[3].steer.setAngle(new Rotation2d());
         setDriveVoltage(v);
       }, null, this)
     );
+  }
 
-    angularRoutine = new SysIdRoutine(
-      new SysIdRoutine.Config(Volts.of(0.5).per(Seconds), Volts.of(3.5), null, (state) -> Logger.recordOutput("SysIdTestState", state.toString())), 
+  private SysIdRoutine setAngularRoutine() {
+    static final double DEFAULT_VX_METERS_PER_SECOND = 0;
+    static final double DEFAULT_VY_METERS_PER_SECOND = 0;
+    static final double DEFAULT_OMEGA_RADIANS_PER_SECOND = 100;
+
+    return new SysIdRoutine(
+      new SysIdRoutine.Config(
+        Volts.of(0.5).per(Seconds), 
+        Volts.of(3.5), 
+        null, 
+        (state) -> Logger.recordOutput("SysIdTestState", state.toString())
+      ), 
       new SysIdRoutine.Mechanism((Voltage v) -> {
-        var states = kinematics.toSwerveModuleStates(new ChassisSpeeds(0, 0, 100));
-        for(int i = 0; i < modules.length; i++) {
-          modules[i].steer.setAngle(states[i].angle);
-        }
-        setDriveVoltage(v);
-      }, null, this)
+          var states = kinematics.toSwerveModuleStates(
+            new ChassisSpeeds(
+              DEFAULT_VX_METERS_PER_SECOND, 
+              DEFAULT_VY_METERS_PER_SECOND, 
```

Comment:
not defaults and probably shouldn't be constantified - they're just placeholder numbers to get the module angles for this specific sysid routine that spins.  i personally think a comment that just states that they're used to get the correct module angles for a spinning robot would suffice?

Reply:
Maybe initial would be better? Since it's not a default.

Reply:
that name isnt even correct either though, cause all that's doing is asking the kinematics to give us the module angles for a robot that should spin, so we just pick some placeholder ChassisSpeed that represents pure rotational motion, if that makes any sense. we don't use the speed the kinematics outputs at all. it's just an alternative to
```java
modules[0].steer.setAngle(Rotation2d.fromDegrees(45));
modules[1].steer.setAngle(Rotation2d.fromDegrees(135));
modules[2].steer.setAngle(Rotation2d.fromDegrees(-45));
modules[3].steer.setAngle(Rotation2d.fromDegrees(45));
```
or whatever the angles are. (the sysid routine itself picks the voltages)

Reply:
Maybe call it ALT for alternative? We need something to replace the constants - could also remove it entirely but then it would just be a generic name and not reflective of what it actually is